butane_cli/src/main.rs

```diff
@@ -73,8 +73,9 @@ However if the migration has been manually edited, it will need to be manually r
     },
     /// Embed migrations in the source code.
     Embed,
-    /// Rollback migrations. With no arguments, undoes the latest migration. If the name of a migration is specified, rolls back until that migration is the latest applied migration.
-    Rollback {
+    /// Undo migrations. With no arguments, undoes the latest migration. If the name of a migration is specified, rolls back until that migration is the latest applied migration.
+    #[command(alias = "rollback")]
+    Unmigrate {
```

Comment:
The alternative here is to adopt the django approach of unmigrate being `migrate <older_migration_name`.
IMO it is better to keep them separate, so the command name clearly communicates that this is a destructive operation.